### DISCUSSION TITLE
Error for wallet connection

### DISCUSSION THREAD

**Discussion by rashidkarim11** [wevm/wagmi#2558] [Q&A]
No QueryClient set, use QueryClientProvider to set one

**Comment by whileMeAwakeMeCode** [comment]
Hi ! 
I got the same error when trying to use `useSignMessage` hook.
I tried to wrap my app with QueryClientProvider with no success.
I also tried to delete and re-install my node_modules folder in case anything happened during install.

I don't get it.
I'm using :
    "wagmi": "^1.2.0"
